Academia-750/academia750-frontend-vue
# DBT-790 clone question page

## Context


Change made is supposed to help an admin to clone or duplicate questions without creating a new question from scratch


[Link To Task](Duplicate Question → Fill create form with an existing question)

## Test







- Login as admin go to topics
- create a question
- after that go to the questions list and click on clone to clone it
- It should create a new question
- 
## Platform


Google chrome

## Proof of Concept
<img width="967" alt="Screenshot 2023-07-31 at 13 42 20" src="https://github.com/Academia-750/academia750-frontend-vue/assets/79788219/11a2d4ff-bdbd-4d2f-8ad0-1ff9ccf034a7">
